src/redux/themeSlice.tsx

```diff
@@ -0,0 +1,27 @@
+import { createSlice } from "@reduxjs/toolkit";
+
+export const getTheme = () => {
+  const theme = localStorage.getItem('theme');
```

Comment:
Maybe call this `themeId` or something similar, to disambiguate it from the theme objects in cssStyles.tsx.